store/tikv/coprocessor.go

```diff
@@ -498,6 +500,13 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 
 		worker.handleTask(ctx, task, respCh)
 		close(task.respChan)
+		if worker.respChan != nil {
+			if worker.sendRate == nil {
+				// unreachable code
+				panic("copIteratorWorker's sendRate should be set when KeepOrder equals false")
+			}
```

Comment:
The code is necessary? When worker.sendRate is nil?

Reply:
The code here should be unreachable. We use panic to make sure worker.sendRate should be init in this case.